pom.xml

```diff
@@ -66,15 +67,30 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>${maven.surefire.plugin.version}</version>
+      </plugin>
+      
     </plugins>
   </build>
 
+  <dependencies>
```

Comment:
If you have this declared as a plugin I don't think you also need it declared as a `dependency`, can you remove this?